lib/default.nix

```diff
@@ -61,10 +61,10 @@ let
       boolToString mergeAttrs flip mapNullable inNixShell min max
       importJSON warn info nixpkgsVersion version mod compare
       splitByAndCompare functionArgs setFunctionArgs isFunction;
-    inherit (fixedPoints) fix fix' extends composeExtensions
+    inherit (fixedPoints) fix fix' converge extends composeExtensions
       makeExtensible makeExtensibleWithCustomName;
     inherit (attrsets) attrByPath hasAttrByPath setAttrByPath
-      getAttrFromPath attrVals attrValues catAttrs filterAttrs
+      getAttrFromPath attrVals attrValues getAttrs catAttrs filterAttrs
```

Comment:
fwiw, not all new functions need to be exposed here.

Reply:
(this is not, explicitly, a request to change what you've done)

Reply:
Good to know. (Although I've wanted `getAttrs` on several occasions, and `converge` I think has a unique enough name, that I think it's warranted for both in this case.)